### DISCUSSION TITLE
The future of SocioBoard

### DISCUSSION THREAD

**Discussion by vaughngx4** [socioboard/Socioboard-5.0#391] [General]
In the true spirit of open source and the open source community I would like to pose the question - what is happening with this project?

In the short time I have been watching/contributing to this project I've seen 1 update(most of which was my code) and 1 response from the developers. I am impressed with the code that is there but the "open source" aspect leaves much to desire in terms of an open source project. Am I missing something? Is there a non-english repo of the source? Has the project been abandoned?

The SSO versions of SocioBoard have unresolved certificate issues which I did try to bring to their attention. Which also makes me believe that this was a side project that has since lost momentum.

Where to from here? I personally don't have the time to dedicate to fix everything, or at least attempt to understand everything that there is little to no documentation for.

Has anyone ever gotten their own social API functions working? 

I again pose the question - where to from here?

**Comment by julien-garrigue** [comment]
Hi!
The website [sociobo](https://socioboard.org/) seems to be down.
Does the open source project is droped?

**Reply by vaughngx4** [reply]
My security blocked it... Not going to question why. I started a new project over at https://github.com/vaughngx4/OpenSMM that's coming along pretty nicely.

**Reply by OllySalanson** [reply]
@julien-garrigue The site seems to be hosted at [https://socioboard.com/](https://socioboard.com/)

**Reply by julien-garrigue** [reply]
> @julien-garrigue The site seems to be hosted at https://socioboard.com/

Thanks but I'm talking about the open source project...

**Reply by AnselmBilbo** [reply]
is there any working link for the open source project? socioboard.com is currently down, but i can't find a comment about it.
